### PR TITLE
Move messages up to sidebar on the right which can be toggled

### LINKED ISSUE #823: Messages: button and "right-side" messages
## Description

We want the messages feature to be immediately findable for users. It is located on the record pages in Describe, which also contains the object's record and provenance information. Moving the messages to another page may be too complicated at this time, but having it with the other information can be confusing with everything else on the page. It was suggested we move the messages to the right-side of the record and have a button that would allow messages to be made visible or moved from view. 

## Value added

The suggestion to have the messages appear at the right-side of the record would be useful for users to be able to compare information in the messages with the record itself. For example, some messages from a curator may suggest changes to a specific field in the record, such as the "Description" field. The submitter may see what they have written on the left and quickly comment back to the curator to receive feedback for a change they might make.

Having the messages appear or disappear is useful for the different purposes the record is being used. For example, messages are needed for communication during the curation process, but they may not be needed if a moderator is just checking a record for remediation or analytic purposes. 

## Image shows the addition of a blue "messages" button
Screen Shot 2023-01-06 at 10 24 14 AM

## Image shows the messages appear after the blue "messages" button is selected and would disappear is selected again
Screen Shot 2023-01-06 at 10 29 14 AM

## Acceptance Criteria
- [ ] A bright blue button is added for the messages 
- [ ] Messages will appear (at right) when button is selected and disappear when it is selected again
- [ ] The message input box is still below the messages (include a "submit" button)
- [ ] The record information fields may need to move from the right side of the screen for this feature to work properly

### PR DESCRIPTION
**Pull request by mccalluc**
- Fix #823 

I'm a little uncomfortable with more ad-hoc jquery, but I think this is what is requested.

Defaults to open:
Screen Shot 2023-01-24 at 3 47 40 PM

Can be closed:
Screen Shot 2023-01-24 at 3 47 52 PM

### CODE REVIEW

**Comment by mccalluc** [comment]
I should add tests. Back to draft.

File: app/assets/stylesheets/components/_container.scss [hunk 1]
```diff
@@ -19,7 +19,6 @@
   .btn-primary {
     background-color: #f57e21;
     border-color: #914810;
-    margin: 20px 0px;
```
Comment (mccalluc) [inline app/assets/stylesheets/components/_container.scss:22]: This style on one button was giving the whole button bar more height, but it didn't align with the new column added in this PR.

File: app/views/works/_work_activity_messages.html.erb [hunk 2]
```diff
@@ -1,5 +1,4 @@
 <div>
-  <h2>Messages</h2>
```
Comment (mccalluc) [inline app/views/works/_work_activity_messages.html.erb:2]: Not in mockup: The button above may be enough of a label.

File: spec/system/work_create_spec.rb [hunk 3]
```diff
@@ -109,11 +109,17 @@
...
       visit(work_path(Work.last))
 
+      has_been_created_message = "#{title} has been created"
       within("ul.work-messages") do
-        expect(page).to have_content("#{title} has been created")
+        expect(page).to have_content(has_been_created_message)
         expect(page).to have_content("#{title} is ready for review")
       end
 
+      click_on "Hide Messages"
```
Comment (carolyncole) [inline spec/system/work_create_spec.rb:118]: ❤️

**Review by carolyncole — APPROVED** [review]

**Pull request merged**